docs/backends.md

```diff
@@ -308,3 +308,63 @@ data:
   current-password-again: <path:projects/12345678987/secrets/test-secret#password#latest>
   password-old: <path:projects/12345678987/secrets/test-secret#password#another-version-id>
 ```
+
+### AZURE Key Vault
+
+##### Azure Authentication
+Refer to the [Use environment-based authentication](https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authorization#use-environment-based-authentication) in the Azure SDK for Go.
+
+Any secrets that are disabled in the key vault will be skipped if found.
+
+Please note that for Azure, `path` is the unique name of your key vault.
+
+These are the parameters for Azure:
+```
+AVP_TYPE: azurekeyvault
+```
+
+##### Examples
+
+###### Path Annotation
+
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: test-secret
+  annotations:
+    avp.kubernetes.io/path: "keyvault"
+type: Opaque
+data:
+  password: <test-secret>
+```
+
+###### Inline Path
+
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: test-secret
+type: Opaque
+data:
+  password: <path:keyvault#test-secret>
+```
+
+###### Versioned secrets
+
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: test-secret
+  annotations:
+    avp.kubernetes.io/path: "keyvault"
+    avp.kubernetes.io/secret-version: "8f8da2e06c8240808ee439ff093803b5"
```

Comment:
If we are going to use versions, since we established that the annotation version doesn't make sense, I would say lets remove that from the example and then in the code lets ignore that annotation (It actually looks like we are already ignoring) and then add a note that says versioning only available for inline paths.

Reply:
Yeah, its really confusing doing the above with how this is working. Let me add that to the documentation.

Reply:
Docs updated, hopefully its OK now.